website/content/en/pre-docs/deploy-guide/_index.md

```diff
@@ -0,0 +1,165 @@
+---
+title: "Deployment Guide"
+linkTitle: "Deployment Guide"
+weight: 50
+---
+
+Introducing new software the modifies how Kubernetes operates is complex, and no single solution exists. 
+
+This guide describes how Karpenter interacts with external resources, and modifies cluster behavior. 
+
+## Resources Dependencies
+
+### Cluster on AWS
+
+Currently, Karpenter creates nodes (EC2 Instances) through the AWS cloud provider module, the only such module implemented. Clusters on other cloud providers are contemplated, but are not currently supported. 
+
+Theoretically, a control plane could be hosted separately from Karpenter provisioned AWS instances. However, this complicates identity and authorization substantially. The Karpenter controller (i.e., agent) would need authorization with AWS APIs to provision new instances, and permissions with the control plane to add the new nodes. 
+
+Additionally, the intricacies of securing control plane communication between clouds presents another significant problem. 
+
+The getting started guide covers [configuring EKS to accept newly provisioned instances as nodes](../getting-started/#create-the-karpenternode-iam-role) (see, *KarpenterNodeRole* and *system:bootstrappers*). If using a self-managed (or alternatively provided) control plane, this must be manually handled. Consult the [*User Data - Autoconfigure* section](../cloud-providers/aws/launch-templates/#user-data---autoconfigure) of *Launch Templates* for more information on joining nodes to a cluster.
+
+### Minimum Kubernetes Version
+
+Karpenter supports the [same set of Kubernetes versions as EKS supports](https://docs.aws.amazon.com/eks/latest/userguide/kubernetes-versions.html) for new clusters. Karpenter strongly encourages a posture of promptly adopting new versions of Kubernetes.
+
+### Service Account
+    - `Karpenter` Service Account
+
+The `Karpenter` service account needs sufficient cluster permissions to, for example, add new nodes and bind pods to nodes. Review the ["karpenter-controller" `role` and `ClusterRole`](rbac.yaml) from the default helm chart to understand what permissions the Karpenter service account needs. The Karpenter helm chart includes other roles (`karpenter-webhook`), but the "karpenter-controller" resources require uniquely broad access.

```

Comment:
Let's point to https://github.com/awslabs/karpenter/blob/main/charts/karpenter/templates/controller/rbac.yaml to avoid needing to update the file in multiple places.

Reply:
agreed